lmnet/lmnet/datasets/mscoco.py

```diff
@@ -198,7 +198,7 @@ def coco(self):
     def _image_ids(self):
         """Return all files and gt_boxes list."""
         classes = [class_name for class_name in self.classes if class_name is not "__background__"]
-        target_class_ids = self.coco.getCatIds(catNms=classes)
+        target_class_ids = set(self.coco.getCatIds(catNms=classes))
```

Comment:
why do you use `set()` here?

Reply:
@tsawada I have 2 reasons.

When another person looks at the code in the future, it can be recognized that the same description is written.
If they can be recognized as the same, it is easy to fixed or abstracted.
That's why I used `set`.

It's hard to tell if you're using `self.classes` in this class or if you're using looped `classes`.
I want to distinguish each one.
But, since the scope of the fixes is getting wider and wider, I have only described `set`.

Reply:
I think we don't need set() here. Please leave it as a list.